---
sensor_msgs/Image crop_color_img
sensor_msgs/Image crop_depth_img
